Solutions/Azure Key Vault/Analytic Rules/KeyVaultSensitiveOperations.yaml

```diff
@@ -32,16 +32,16 @@ query: |
   | where requestUri_s !~ "None" and isnotempty(requestUri_s)
   | where ResourceType =~ "VAULTS" and ResultType =~ "Success"
   | where OperationName in~ (SensitiveOperationList)
-  | summarize EventCount=count(), StartTimeUtc=min(TimeGenerated), EndTimeUtc=max(TimeGenerated), TimeTriggered=makelist(TimeGenerated),OperationNameList=make_set(OperationName), RequestURLList=make_set(requestUri_s), CallerIPList = make_set(CallerIPAddress),  CallerIPMax= arg_max(CallerIPAddress,*) by ResourceType, ResultType, Resource, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, clientInfo_s
-  | extend timestamp = StartTimeUtc, IPCustomEntity = CallerIPMax, AccountCustomEntity = identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g
+  | summarize EventCount=count(), StartTimeUtc=min(TimeGenerated), EndTimeUtc=max(TimeGenerated), TimeTriggered=make_list(TimeGenerated),OperationNameList=make_set(OperationName), RequestURLList=make_set(requestUri_s), CallerIPList = make_set(CallerIPAddress),  CallerIPMax= arg_max(CallerIPAddress,*) by ResourceType, ResultType, Resource, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, clientInfo_s
+  | extend timestamp = StartTimeUtc
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: ObjectGuid
+        columnName: identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g
```

Comment:
Lets map identity_claim_upn_s here which has name and upn suffix which is strongest identifier. and change identifier to FullName

Reply:
Done